src/tests/components/App.test.js

```diff
@@ -0,0 +1,36 @@
+import { render } from "@testing-library/react";
+import axios from "axios";
+import App from "../../components/App";
+
+describe("App", () => {
+  it("renders correctly", async () => {
```

Comment:
I see what you wanted to do here, but if you look at the snapshot there is no data being displayed, just the bare bones of the component (which is all you really need to test in a snapshot). You can check this is the case by changing the test to just 
```
  it("renders correctly", () => {
    const { asFragment } = render(<App />);
    expect(asFragment()).toMatchSnapshot();
  });
```
and re running it. If the data was being used, you would expect the snapshot to be different from the previous one and therefore the test would fail, but that will still pass.
The axios mock method here is more used to assert the values passed back from the mock are used in the component, but you dont need to do this for snapshot testing.
There is a way to do this but its a bit more advanced so we dont tech it as standard, just want to cover the basics for now